doc/installing.rst

```diff
@@ -94,8 +94,7 @@ Some users have run into the following complications when installing the
    environmental variables to your shell rc that point to the reverted
    directory structure.
  
- - At the moment, an implicit dependency on PyQt5 remains (in addition to
-   PySide2) because of our dependence on ``linetools``.
+ - At the moment, an implicit dependency on PySide2 remains because of our dependence on ``linetools``.
```

Comment:
last i checked `linetools` still required both PySide2 and PyQt5 to be installed in order to work.

Reply:
There was a recent PR to line tools: https://github.com/linetools/linetools/pull/526/files

Reply:
Hmm could you let me know which version that was? PypeIt's requirements.txt was pinning an old linetools version (that required PyQt5) for quite a while (0.3dev2147) but a newer version of linetools (0.3dev2227) should require only PySide2 and the head of linetool's git repo should now only require QtPy. Once a new linetools version is released on pypi that only depends on QtPy, I'll update the language here.

But more generally, I probably need to take a deep look at the install and dev install docs and do a bit of rewriting.

Reply:
Thanks @finagle29. We definitely need to remove the PySide2 dependency implicit or explicit since that is at the heart of all the installation woes. If users want to use it and can handle installing it, then they can as per the ginga approach.